app/controllers/lists_controller.rb

```diff
@@ -0,0 +1,41 @@
+class ListsController < ApplicationController
+  def index
+    @lists = List.all
+  end
+
+  def create
+    @list = List.new(list_params)
+    @list.save
+    redirect_to root_path
+  end
+
+  def new
+    @list = List.new
+  end
+
+  def edit
+    @list = List.find(params[:id])
```

Comment:
Thoughts on extracting a `find_list` method? That way if you ever change how a list is found in the future, you can do it in one place instead of 4.